core/services/ocr2/plugins/directrequestocr/reporting.go

```diff
@@ -34,6 +36,43 @@ type functionsReporting struct {
 
 var _ types.ReportingPlugin = &functionsReporting{}
 
+var (
+	promReportingPlugins = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "functions_reporting_plugin_instances",
+		Help: "Metric to track number of reporting plugin instances",
+	}, []string{"jobID"})
+
+	promReportingPluginsQuery = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "functions_reporting_plugin_query",
+		Help: "Metric to track number of reporting plugin Query calls",
+	}, []string{"jobID"})
+
+	promReportingPluginsObservation = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "functions_reporting_plugin_observation",
+		Help: "Metric to track number of reporting plugin Observation calls",
+	}, []string{"jobID"})
+
+	promReportingPluginsReport = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "functions_reporting_plugin_report",
+		Help: "Metric to track number of reporting plugin Report calls",
+	}, []string{"jobID"})
+
+	promReportingAcceptReports = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "functions_reporting_plugin_accept",
+		Help: "Metric to track number of accepting reports",
+	}, []string{"jobID"})
+
+	promReportingTransmitReports = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "functions_reporting_plugin_transmit",
+		Help: "Metric to track number of transmiting reports",
+	}, []string{"jobID"})
+
+	promReportingTransmitBatchSize = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "functions_reporting_plugin_transmit_batch_size",
```

Comment:
Same concern as above about a sum of batch sizes - is that how it works?

Reply:
See above.